### PR TITLE
issues-1

### PR DESCRIPTION
**Pull request by payamST**

### CODE REVIEW

**Review by serialcool — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -1,42 +1,23 @@
...
+        <groupId>com.ibmix.magkit</groupId>
+        <artifactId>magkit-pom</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
     <artifactId>magnolia-vanity-url</artifactId>
     <version>1.6.1-SNAPSHOT</version>
     <packaging>jar</packaging>
     <name>magnolia-vanity-url Magnolia Module</name>
-
-    <inceptionYear>2013</inceptionYear>
```
Comment (jfrantzius) [inline pom.xml:11]: I think we should keep `inceptionYear` separate in each Magkit module, so let's not delete this here

Comment (payamST) [reply pom.xml:11]: added again

File: pom.xml [hunk 2]
```diff
@@ -1,42 +1,27 @@
...
-        <source.javaversion>11</source.javaversion>
-        <target.javaversion>11</target.javaversion>
-
-        <source.encoding>UTF-8</source.encoding>
-        <project.build.sourceEncoding>${source.encoding}</project.build.sourceEncoding>
-
-        <!-- checkstyle can be disabled with this property -->
-        <checkstyle.skip>false</checkstyle.skip>
-        <checkstyle.config>src/config</checkstyle.config>
+        <checkstyle.config>com/de/ibmix</checkstyle.config>
```
Comment (jfrantzius) [inline pom.xml:18]: do we need this property here, or can it be set in magkit-pom?

Comment (payamST) [reply pom.xml]: removed

**Pull request merged**